tests/variantstudy/model/command/test_remove_cluster.py

```diff
@@ -13,6 +15,7 @@
 
 
 class TestRemoveCluster:
+    @pytest.mark.parametrize("empty_study", ["empty_study_720.zip", "empty_study_870.zip"], indirect=True)
```

Comment:
It could be good to add an integration test to ensure the endpoint delete cluster properly works.

Reply:
Done